### PR TITLE
Move CategoryMerchandising util to php-sdk

### PR DESCRIPTION
**Pull request by dairbuirabass**
## Description
Moved CategoryMerchendasing utility from CMP to php-sdk module. PR: https://github.com/Nosto/nosto-php-sdk/pull/319

## Related Issue
Closes 136

## Motivation and Context
The functionality within utility is more suited to PHP-SDK module than CMP

## How Has This Been Tested?
Locally.

## Checklist:
- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] I have assigned the correct milestone or created one if non-existent.
- [x] I have correctly labeled this pull request.
- [x] I have linked the corresponding issue in this description.
- [ ] I have updated the corresponding Jira ticket.
- [x] I have requested a review from at least 2 reviewers
- [x] I have checked the base branch of this pull request
- [x] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

**Comment by dairbuirabass** [comment]
@olsi-qose Inspections fail due to parallel changes in the php-sdk module, what's best way to handle this?

**Comment by olsi-qose** [comment]
> 

Make the changes in the PHP-SDK repo first and release them. Then bump the sdk dependency version here

**Comment by dairbuirabass** [comment]
> > 
> 
> Make the changes in the PHP-SDK repo first and release them. Then bump the sdk dependency version here

https://github.com/Nosto/nosto-php-sdk/pull/319 here are the changes

File: Utils/CategoryMerchandising.php [hunk 1]
```diff
@@ -46,19 +44,4 @@ class CategoryMerchandising
     const DISPATCH_EVENT_KEY_RESULT = 'result';
     const DISPATCH_EVENT_KEY_LIMIT = 'limit';
```
Comment (olsi-qose) [inline Utils/CategoryMerchandising.php:45]: Move these constants to other classes, like Pre/PostRequestAction and delete this class

**Review by olsi-qose — CHANGES_REQUESTED** [review]

**Review by olsi-qose — APPROVED** [review]

**Pull request merged**